src/Controls.js

```diff
@@ -3,55 +3,58 @@ import './controls.css';
 
 class Controls extends Component {
   render() {
+    const detailState = localStorage.getItem('userMoves') == null ? true : false;
```

Comment:
rename it to `isDetailsExpanded`. It should not have 'state' in its name because it's not a React state, it is a computed value.